### PR TITLE
Edit documentation

### PR DESCRIPTION
**Pull request by yangc95**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -112,11 +112,11 @@ To link your Codecademy user profile to GitHub:
 4. Select the Github icon under "Or log in using:"
 5. Proceed through the OAuth process.
 
-## Any tips for a Pull Request (PR)?
+## Any necessary tips for a Pull Request (PR)?
```
Comment (yangc95) [inline .github/CONTRIBUTING.md:115]:
```suggestion
## Any requirements for a Pull Request (PR)?
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -112,11 +112,11 @@ To link your Codecademy user profile to GitHub:
...
 
 - Before making a PR, make sure you pushed your changes from a branch other than `main`.
 - Name the new branch after the changes being pushed to the PR.
-- Keep your PRs byte-sized. No more than 3 new entries per PR!
+- Keep your PRs byte-sized. No more than 3 new entries per PR! And no more than 3 PRs open at a time!
```
Comment (KTom101) [inline .github/CONTRIBUTING.md:119]: There have been cases where contributors were waiting for second reviews and opened more PRs after being assigned to new issues. So I'm thinking "no more than 3 PRs" wouldn't always apply.

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (KTom101) [reply .github/CONTRIBUTING.md:115]: I like "requirements" better.

**Review by KTom101 — COMMENTED** [review]
@yangc95 Left a couple of comments

**Review by SSwiniarski — APPROVED** [review]
@yangc95 Looks good.

**Pull request merged**